uiowa/uiowa
# Remove empty config directories.

Merged into develop and deployed.

### Testing
Resolves #4095 
- Inspect task ID 147403437 output on uiowa.dev and look at the logs for accreditation.uiowa.edu which had an empty config directory removed. There should be no errors/warnings. 
- Check the status report on https://accreditation.dev.drupal.uiowa.edu/admin/reports/status. There should be no errors/warnings related to the config directory.

### Testing
Resolves #4096
- Run `blt umc foo.com --no-commit --no-db` and see that no empty folder is created in the `config` directory. Review the `config/README.md` site split docs. Discard all changes afterwards.
